docker/Install-ChocolateyInContainer.ps1

```diff
@@ -72,13 +72,13 @@ param (
   # Call chocolatey install
   Write-Output "Installing chocolatey on this machine"
   $toolsFolder = Join-Path $tempDir "tools"
-  $chocInstallPS1 = Join-Path $toolsFolder "chocolateyInstall.ps1"
+  $chocoInstallPS1 = Join-Path $toolsFolder "chocolateyInstall.ps1"
 
-  & $chocInstallPS1
+  & $chocoInstallPS1
 
   Write-Output 'Ensuring chocolatey commands are on the path'
-  $chocInstallVariableName = "ChocolateyInstall"
-  $chocoPath = [Environment]::GetEnvironmentVariable($chocInstallVariableName)
+  $chocoInstallVariableName = "ChocolateyInstall"
+  $chocoPath = [Environment]::GetEnvironmentVariable($chocoInstallVariableName)
   if ($chocoPath -eq $null -or $chocoPath -eq '') {
     $chocoPath = 'C:\ProgramData\Chocolatey'
   }
```

Comment:
I can't hang a comment off a binary file change. the `.doc` file change is:

```diff
-If this is set before calling Merge, then it specifies the path and filename to an atttribute assembly, an assembly that will be used to get all of the assembly-level attributes such as Culture, Version, etc. 
+If this is set before calling Merge, then it specifies the path and filename to an attribute assembly, an assembly that will be used to get all of the assembly-level attributes such as Culture, Version, etc.
```
~
```diff
-atttribute
+attribute
```

Unfortunately, the tool I used to make the edit this file (TextEdit.app) converted the file from utf8 (?) to utf16 (?) and thus I can't spell check it further.  I'm not sure what your requirements are for this file, but I imagine someone else could properly edit it w/ the original document editor if one wanted to retain its original file format.